ods_ci/tests/Resources/Page/ODH/JupyterHub/JupyterHubSpawner.robot

```diff
@@ -354,6 +354,12 @@ Spawn Notebook With Arguments  # robocop: disable
             Run Keyword And Warn On Failure   Login To Openshift  ${username}  ${password}  ${auth_type}
             ${authorization_required} =  Is Service Account Authorization Required
             IF  ${authorization_required}  Authorize jupyterhub service account
+
+            # For Jupyter 4, we need to update global default variable values (images 2024b and newer)
+            # This calls method from JupyterLibrary Version.resource module
+            # TODO - shall be uncommented once the 2024b images will land into the product
+            # IF  "${version}"=="default"  Update Globals For JupyterLab 4
```

Comment:
If the workbench pod is already running and you want to know the  jupyterlab version you could use keyword `Run Command In Container` and run this command `oc exec -n ${namespace}  ${pod_name} -- jupyter --version | grep jupyterlab | awk '{print $3}'`

Reply:
Thank you Jorge. That is correct. I was a bit lazy to do it that way. I presume that having the check for `default` value should be sufficient for now and eventually won't be needed once we move further with another upgrade in 6 months. :slightly_smiling_face: